### PR TITLE
feat: CLIN-687 - add spark-jobs + cleanup-jobs

### PR DESCRIPTION
**Pull request by creativeyann17**
- [x] fix QA delete ES indexes clean-job
- [x] add staging clean-jobs
- [x] add staging spark-jobs
- [x] release staging etl-variant
- [x] release staging etl-hpo

### CODE REVIEW

File: staging/apps/cleanup-jobs/rolling-es-indices/rolling-es-indices.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: batch/v1
```
Comment (creativeyann17) [inline staging/apps/cleanup-jobs/rolling-es-indices/rolling-es-indices.yml:2]: do we need one like this for staging ?

File: staging/apps/spark-jobs/fhir-export-etl/job.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                  name: s3-etl-credentials
+                  key: S3_SECRET_KEY
+            - name: AWS_ENDPOINT
+              value: https://s3.ops.cqdg.ferlab.bio
+            - name: AWS_DEFAULT_REGION
+              value: regionone
+            - name: AWS_BUCKET_NAME
+              value: clin-staging-app-datalake
+            - name: AWS_PREFIX
+              value: $COLOR
```
Comment (creativeyann17) [inline staging/apps/spark-jobs/fhir-export-etl/job.yml:46]: TODO

File: staging/apps/spark-jobs/file-import-etl/job.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                  name: s3-files-processing-credentials
+                  key: S3_SECRET_KEY
+            - name: AWS_ENDPOINT
+              value: https://s3.ops.cqdg.ferlab.bio
+            - name: AWS_DEFAULT_REGION
+              value: regionone
+            - name: AWS_BUCKET_NAME
+              value: clin-staging-app-files-import
+            - name: AWS_PREFIX
+              value: $COLOR
```
Comment (creativeyann17) [inline staging/apps/spark-jobs/file-import-etl/job.yml:46]: TODO

File: staging/apps/spark-jobs/enriched-etl/job.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    spec:
+      serviceAccountName: spark
+      containers:
+        - name: enriched-etl-$JOBNAME
+          image: ferlabcrsj/spark:3.1.2
+          env:
+            - name: SPARK_CLASS
+              value: bio.ferlab.clin.etl.enriched.RunEnriched
+            - name: SPARK_JAR
+              value: s3a://clin-staging-app-datalake/spark/clin-variant-etl.jar
```
[the remark this answers is not in the document]
Comment (jecos) [reply staging/apps/spark-jobs/enriched-etl/job.yml:17]: looks like the jar file does not exist yet. but it should. we need to check if clin-variant-etl project push the jar in this bucket whent master is tagged

File: staging/apps/spark-jobs/file-import-etl/job.yml (hunk 3 above)
Comment (jecos) [reply staging/apps/spark-jobs/file-import-etl/job.yml:46]: you can simply set this env var to `files`

File: staging/apps/spark-jobs/fhir-export-etl/job.yml (hunk 2 above)
Comment (jecos) [reply staging/apps/spark-jobs/fhir-export-etl/job.yml:46]: This env var is not used in this job. You can remove it

**Review by jecos — CHANGES_REQUESTED** [review]

**Review by jecos — APPROVED** [review]

**Pull request merged**